poetry/core/masonry/builders/builder.py

```diff
@@ -191,7 +195,9 @@ def find_files_to_add(
         if self._package.build_script and not exclude_build:
             to_add.add(
                 BuildIncludeFile(
-                    path=self._package.build_script, source_root=self._path
+                    path=self._package.build_script,
+                    project_root=self._original_path,
+                    source_root=self._path,
```

Comment:
Is original path and path ever different?

Reply:
At least it was the case for the `CompleteBuilder` which we have removed some days/weeks ago. There the `sdist` was unpacked somewhere else and for checking what's excluded by git, we needed to know where the original path was. I can remove it, but not sure if we get problems anywhere else :thinking:

Reply:
Tests passes when I remove `self._original_path` and replace it in the code with `self._path`